### DISCUSSION TITLE
Freeplane Portable being blocked after update

### DISCUSSION THREAD

**Discussion by olazzzzz** [freeplane/freeplane#1667] [Help]
Hi everyone,

I use Freeplane extensively for my work, and could not manage without it. As the organization I work for does not allow installation of software on the laptop, I use it from a portable USB.
Everything was working great, the thing is that after I last updated Freeplane from version 1.11.05 to 1.11.10 (together with Java updates and of the portable platform itself), now Freeplane does not launch because it is being blocked.
Is there a way around it? And can it be maintained also upon updating?
Thank you for your help!

**Comment by dpolivaev** [comment]
What do you mean by "being blocked"? What or who blocks it and what does it look like?

**Reply by olazzzzz** [reply]
I mean that the organization (my employer) blocks freeplane from opening (from the USB portableapps).
It looks like a pop-up on the left lower corner that reads: "Action blocked. Your administrator caused Windows Security to block this action. Contact your helpdesk."

**Reply by dpolivaev** [reply]
Could you clarify why they blocked it?

**Reply by olazzzzz** [reply]
They have a policy of only allowing pre-installed (mostly microsoft) software on working PCs, so anything that you ask the IT to install will be rejected, and if you try to install yourself will most likely be blocked by a firewall. This is done for security reasons, but in the case of a software like freeplane I think it is just absurd.

**Reply by dpolivaev** [reply]
Well, they have not blocked earlier versions, right?
On my end, I changed the implementation of the launcher (freeplane.exe) so that it can be used with Java versions from 8 to 21. I do not know if they have any concerns.

**Reply by olazzzzz** [reply]
The earlier version I was using, 1.11.5 was working well, provided a little tweaking was done on the USB to work from a Temp directory *( as explained here https://portableapps.com/node/35146).
Do you think that perhaps that the new launcher is causing the firewall to react? Should I revert to the older
 version?

**Reply by dpolivaev** [reply]
I have no clue. Ask your admins.

**Reply by WeirdAnalyst** [reply]
Could be that your admins have blocked execution of software from usb. Try copying to hard drive and then execute again. You have to also copy java on HD and set env variable in order for freeplane launcher to locate java.exe

**Reply by olazzzzz** [reply]
Thank you for your answers. 
I just reinstalled version 1.11.05 in my USB and that one works (meaning it launches and works as expected). I am guessing that the newer FP version triggered some firewall protections, perhaps by the new behavior in the launcher (?) but the usb is still working as seen above.

**Comment by euu2021** [comment]
I use the portable version that comes in a zip, like this `freeplane_bin-1.11.9-pre13.zip`. 
So, I don't need to use portableapps, as explained at https://github.com/freeplane/freeplane/discussions/462#discussioncomment-7180341

Maybe, doing this way improves your situation?

**Reply by olazzzzz** [reply]
Thank you @euu2021, I will try that option soon and give feedback
